src/event_pusher/mod_event_pusher_push.erl

```diff
@@ -66,8 +66,9 @@
 start(Host, Opts) ->
     ?INFO_MSG("mod_event_pusher_push starting on host ~p", [Host]),
 
-    {ok, _} = wpool:start_sup_pool(gen_mod:get_module_proc(Host, ?MODULE),
-                                   gen_mod:get_opt(wpool, Opts, [])),
+    mongoose_wpool:ensure_started(),
+    WpoolOpts = [{strategy, available_worker} | gen_mod:get_opt(wpool, Opts, [])],
```

Comment:
Do we allow to set the `strategy` in the config file? If so this line will result in selecting `available_worker` strategy even if other is specified in the config.

Reply:
I don't believe we do, explicitly

Reply:
All right.